src/app/modules/main/communities/view.nim

```diff
@@ -50,6 +50,8 @@ QtObject:
       discordImportChannelId: string
       discordImportChannelName: string
       discordImportCommunityImage: string
+      discordImportedChannelId: string
```

Comment:
Do we really need a separate property for this? We now have `discordImportChannelId` and `discordImportedChannelId`, easy to confuse imnsho

Reply:
I created the issue to refactoring this :)